Dockerfile

```diff
@@ -0,0 +1,37 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18.16.0-alpine3.18 AS build-frontend
+
```

Comment:
This is a multistage Docker build. It actually constructs 2 containers

1. A container that has a node 18 environment for building the frontend package
2. A container with a python runtime, that imports the static files generated from the build in step 1 (see the COPY --from below)

You can read more at https://docs.docker.com/build/building/multi-stage/